core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -177,7 +201,7 @@ public Put withValues(Collection<Value<?>> values) {
    *
    * @return a map of {@code Value}s
    */
-  public Map<String, Value<?>> getValues() {
+  public Map<String, Optional<Value<?>>> getValues() {
```

Comment:
This change is actually an incompatible change but we can assume this method is only used internally, so I think it's okay.

Reply:
In Scalar DL, `getValues` is not used in `src` code but is used a lot in `test` code, so we might need to think about introducing `getNullableValues()` or something for backward incompatibility?
Let's discuss it later.